components/PlanList/PlanList.js

```diff
@@ -1,28 +1,97 @@
-import { List } from "./PlanList.styled";
+import { List, Title, Heading, Span } from "./PlanList.styled";
+import { useState } from "react";
+
 export default function PlanList({ plans }) {
+  const myPlans = plans.filter((plan) => plan.for === "Me");
+  const otherPlans = plans.filter((plan) => plan.for !== "Me");
+  const [showPlan, setShowPlan] = useState(false);
```

Comment:
As far as I can see `showPlan` contains the index of the plan you want to show. I'd suggest renaming the state to make this clear and initialising it as `null`

```suggestion
  const [activePlanIndex, setActivePlanIndex] = useState(null);
```

Reply:
Thank you for your feedback , i have already changed it